### DISCUSSION TITLE
shared library libjvm.dylib does not contain the JNI_CreateJavaVM symbol.

### DISCUSSION THREAD

**Discussion by bwiggs** [dbeaver/dbeaver#119] [Q&A] [answered]
Just installed dbeaver-community on my mac using Cask.

```
$ cask info dbeaver-community
dbeaver-community: 3.5.6
DBeaver Community Edition
http://dbeaver.jkiss.org/
/opt/homebrew-cask/Caskroom/dbeaver-community/3.5.6 (294 files, 35M)
https://github.com/caskroom/homebrew-versions/blob/master/Casks/dbeaver-community.rb
==> Contents
  Dbeaver.app (app)
```

Install seemed to go ok, but when I launch the app I get the following error:

![screenshot 2015-12-17 11 05 33](https://cloud.githubusercontent.com/assets/7358/11876222/32238c84-a4ae-11e5-9205-75edb8bb619a.png)

```
$ java -version
java version "1.8.0_66"
Java(TM) SE Runtime Environment (build 1.8.0_66-b17)
Java HotSpot(TM) 64-Bit Server VM (build 25.66-b17, mixed mode)
```

**Comment by serge-rider** [comment, marked as answer]
That could happen in case of broken (or not good enough) Java installation.
Try this workaround: http://stackoverflow.com/a/25486012/1912589

**Comment by bwiggs** [comment]
Fixed after installing Java from https://support.apple.com/kb/DL1572?viewlocale=en_US&locale=en_US
